torch/ao/quantization/fx/prepare.py

```diff
@@ -129,6 +130,31 @@
     "output_act_obs_or_fq_ctr": torch.ao.quantization.qconfig._default_quint8_placeholder_qconfig.activation
 }
 
+def _needs_obs_or_fq(prev_output_dtype, prev_output_is_dynamic, cur_target_dtype, cur_target_is_dynamic, reuse_input_obs_or_fq):
```

Comment:
Will this account for something like
"
conv1
slice
view
conv2
"
Here the conv2's input and conv1's output can share observer such that conv1's output node will have quant/dequant op and conv2's input will also have quant/dequant op but their qparams are shared.

Reply:
yes I think so, it should be supported as long "slice" and "view" are added to the list in https://github.com/pytorch/pytorch/pull/99384